### PR TITLE
story/DATA-4746/change-botify-logos

### PR DESCRIPTION
**Pull request by AdeleD**

### CODE REVIEW

File: botify_docs/static/style/css/main.css [hunk 1]
```diff
@@ -400,7 +400,7 @@ html body .main-content .main-content__header .inner-wrapper .main-content__head
 html body .main-content .main-content__header .inner-wrapper .main-content__header-left .developers-logo {
   width: 393px;
   height: 65px;
-  background: url("../images/developers-logo.png") no-repeat;
+  background: url("../images/developers-logo.gif") no-repeat;
```
Comment (ybastide) [inline botify_docs/static/style/css/main.css]:
```
$ file botify_docs/static/style/images/developers-logo.gif
botify_docs/static/style/images/developers-logo.gif: GIF image data, version 89a, 385 x 82
```

:thinking:

Comment (AdeleD) [reply botify_docs/static/style/css/main.css]: ? 
Yes, we want to display a GIF now :wink:

Comment (ybastide) [reply botify_docs/static/style/css/main.css]: See end of line: `385 x 82` :stuck_out_tongue:

Comment (AdeleD) [reply botify_docs/static/style/css/main.css]: Oh you're right! This is probably why I thought the quality of the GIF was bad :roll_eyes:

Comment (AdeleD) [reply botify_docs/static/style/css/main.css]: In fact, even without my mistake, the quality of the first GIF logo I added was bad. 
But I have a new one now (which is beautiful) :sunglasses:

**Pull request merged**